stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -83,10 +84,12 @@ class StacApi:
         converter=update_openapi,
     )
     router: APIRouter = attr.ib(default=attr.Factory(APIRouter))
-    title: str = attr.ib(default="stac-fastapi")
-    api_version: str = attr.ib(default="0.1")
+    title: str = attr.ib(default=os.getenv("STAC_FASTAPI_TITLE", "stac-fastapi"))
```

Comment:
also I think we usually use pydantic settings to take care of the env variable (enable the use of `.env` files too). I think it will be better to do the same here

Reply:
no worry, I think we will use the solution proposed in #499

Reply:
Waiting for 3.0.0 doesn't sound ideal to me...